scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2671,6 +2529,33 @@ s32 BPF_STRUCT_OPS_SLEEPABLE(layered_init)
 			return ret;
 	}
 
+	dbg("CFG: Dumping CPU proxity map");
```

Comment:
Could maybe kick this out to a helper function and initialize the prox map above, but should be good for now as long as the verifier is happy.